cmd/kubelet/app/options/options.go

```diff
@@ -88,7 +85,7 @@ type KubeletFlags struct {
 	config.ContainerRuntimeOptions
 
 	// certDirectory is the directory where the TLS certs are located (by
-	// default /var/run/kubernetes). If tlsCertFile and tlsPrivateKeyFile
+	// default /var/lib/kubelet/pki). If tlsCertFile and tlsPrivateKeyFile
```

Comment:
this commend was pointing to an outdated path.

Reply:
Good catch!